Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1814,8 +1814,8 @@ - (IBAction)miniFormButtonTapped:(id)sender {
         {
             ORKFormItem *item = [[ORKFormItem alloc] initWithIdentifier:@"fqid_text3"
                                                                    text:@"Text C"
+                                                               optional:NO
                                                            answerFormat:format];
-            item.optional = NO;
```

Comment:
Did you want to update all the other form items that use optional property also? Or did you just do a few to show that it is available?